backend/src/Response/Admin/CaptainFinancialSystem/AdminCaptainFinancialDuesResponse.php

```diff
@@ -23,4 +23,8 @@ class AdminCaptainFinancialDuesResponse
     public int $captainId;
     
     public string $captainName;
+
+    public array $paymentsToCaptain;
```

Comment:
When there won't be payments for a captain, this will cause an error because the type is restricted to array.